views/account/signup/index.ejs

```diff
@@ -5,20 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Sign Up</title>
+
+    <!-- Bootstrap links to make it work  -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css" rel="stylesheet" />
   </head>
   <body>
-    <p class="message"><%= error %></p>
-    <form action="/account/signup" method="post">
-      <label for="name">First Name</label>
-      <input type="text" name="first_name" id="first_name" />
-      <label for="last_name">Last Name</label>
-      <input type="text" name="last_name" id="last_name" />
-      <label for="email">Email</label>
-      <input type="email" name="email" id="email" />
-      <label for="password">Password (NOT ENCRYPTED YET!)</label>
-      <input type="password" name="password" id="password" />
-      <button type="submit">Sign Up</button>
-    </form>
-    <a href="/account/login">Log In</a>
+    <section class="vh-100 bg-image"
+    style="background-image: url('/gooncardbackground.jpg');">
+    <div class="mask d-flex align-items-center h-100 gradient-custom-3">
+      <div class="container h-100">
+        <div class="row d-flex justify-content-center align-items-center h-100">
+          <div class="col-12 col-md-9 col-lg-7 col-xl-6">
+            <div class="card bg-dark" style="border-radius: 15px;">
+              <div class="card-body p-5">
+                <h2 class="text-uppercase text-center text-white mb-5"> Create an account</h2>
+  
+                <p class="message"><%= error %></p>
```

Comment:
Error message that blends in on signup page.

Reply:
Really good catch, it should be fixed in the latest commit